data/calculated_parser/functions.py

```diff
@@ -450,7 +459,47 @@ def criticaldepth(depth, lat, temperature, salinity):
     salinity: The salinity (at all depths) (unitless)
     """
 
+    
     speed = sspeed(depth, lat, temperature, salinity)
+
+
+    """
+    #VECTORIZING -- INCOMPLETE
```

Comment:
Large commented-out code blocks should not be put on master @dwayne-hart

Reply:
If this part won't be completed soon, it should be pulled out of this PR and dumped into another branch w/ draft PR.

Reply:
Moved code to an issue instead